fe/fe-core/src/main/java/com/starrocks/catalog/ResourceMgr.java

```diff
@@ -108,17 +133,27 @@ public void createResource(CreateResourceStmt stmt) throws DdlException {
      * <p>1. Overwrite the resource </p>
      * <p>2. Clear cache in memory </p>
      */
-    public void replayCreateResource(Resource resource) {
+    public void replayCreateResource(Resource resource) throws DdlException {
+        if (resource instanceof HiveResource || resource instanceof HudiResource) {
+            GlobalStateMgr.getCurrentState().getHiveRepository().clearCache(resource.getName());
```

Comment:
Do you still use HiveMetaCache in this version?

Reply:
No, I'll remove the HiveMetaCache in next pr